cairo-sys-rs/src/lib.rs

```diff
@@ -742,6 +747,22 @@ extern "C" {
     pub fn cairo_text_cluster_allocate(num_clusters: c_int) -> *mut TextCluster;
     pub fn cairo_text_cluster_free(clusters: *mut TextCluster);
 
+    #[cfg(any(feature = "freetype", feature = "dox"))]
+    pub fn cairo_ft_font_face_create_for_ft_face(
+        face: FT_Face,
+        load_flags: c_int,
+    ) -> *mut cairo_font_face_t;
+    #[cfg(any(feature = "freetype", feature = "dox"))]
+    pub fn cairo_ft_font_face_create_for_pattern(pattern: *mut FcPattern) -> *mut cairo_font_face_t;
+    #[cfg(any(feature = "freetype", feature = "dox"))]
+    pub fn cairo_ft_font_option_substitute(
+        options: *const cairo_font_options_t,
+        pattern: *mut FcPattern,
+    );
```

Comment:
And if we go for the `freetype` crate then the others can also be added to the non-`sys` bindings :)

Reply:
Potentially, yes.

Reply:
Want to give it a try? Otherwise let's get this in, I guess :)

Reply:
I'll have a go.

Reply:
Edit: Actually lets finish this PR without. They can always be added later, and it keeps the complexity of this PR lower.